game.js

```diff
@@ -42,22 +56,36 @@ function loop() {
         shipSpeed = 0;
         shipPosition = 0;
     }
-    layout();
+    if(shipSpeed >0 &&
+        shipPosition > 0)
+        {
+            layout();
+        }
+        else{
+            document.getElementById('fuel').innerHTML = 'UFO Crashed';
```

Comment:
Note that this shouldn't be in the flame feature branch, it's also not quite the right logic for deciding if it crashed.  You can keep it for now, and fix it when you work on the crash feature